src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -63,20 +67,38 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(lectureEntity);
-
-    // TODO: Add redirect link to lecture site.
+    response.sendRedirect("/index.html");
```

Comment:
Maybe this should be turned into a constant?  Also, we can redirect to just `/` to keep things cleaner

Reply:
fixed